components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -158,6 +176,67 @@ define([
         },
 
         /**
+         * Generates alive, alive and well and deceased radio button section
+         *
+         * @method generateAliveWell
+         * @return {Raphael.el} Raphael element set
+         */
+        generateAliveWell: function() {
+            var node = this.getNode();
+            var lifeStatus = node.getLifeStatus();
+            var aliveandwellStatus = node.getAliveAndWell();
+            //generate 3 radio buttons
+            var labeles = ["Alive", "Alive & Well", "Deceased"];
+            var ticks = [];
+            var rects = [];
+            var height = this.getY()+this._height-this.getHoverBoxHeightExtension()-10;
+            var yTickIndex = 0;
+            var aliveAndWell = editor.getPaper().set();
+
+            for (var i = 0; i < 3; i++) {
+                var circle = this.generateCircle(this.getX()+15, height+20*i, false);
+                if ((lifeStatus == "alive" && !aliveandwellStatus && i == 0)
```

Comment:
That looks like a translatable label, whixh won't work nicely when adding support for other languages.